docs/articles/clustering/member-roles.md

```diff
@@ -51,8 +53,20 @@ akka {
 }
 ```
 
+```csharp
+	var sharding = ClusterSharding.Get(system);
+    var shardRegion = await sharding.StartAsync(
+    typeName: "customer",
+    entityPropsFactory: e => Props.Create(() => new Customer(e)),
+    settings: ClusterShardingSettings.Create(system).WithRole("worker"),
+    messageExtractor: new MessageExtractor(10));
+
+```
+
 ### `DistributedPubSub`
 
+Start the mediator on members tagged with this role. All members are used if undefined or empty.
```

Comment:
Mediator will send messages to all nodes tagged with this role, otherwise all members will receive Mediator gossip.